data/organizers.json

```diff
@@ -14,6 +14,15 @@
   "instagram": "myles",
   "github": "myles",
   "photo": "myles-braithwaite.png"
+}, {
+  "first_name": "Dorian",
+  "last_name": "Puła",
+  "title": "Software Development Engineer",
+  "company": "Points",
+  "bitbucket": "https://bitbucket.org/dorianpula/",
```

Comment:
Could you change this to be just your username.

Reply:
Sure!  I was just experimenting with how the links were rendered in the preview.